core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -476,6 +477,17 @@ public ClientFactoryBuilder tlsAllowUnsafeCiphers(boolean tlsAllowUnsafeCiphers)
         return this;
     }
 
+    /**
+     * The {@link TlsEngineType} that will be used for processing TLS connections.
```

Comment:
```suggestion
     * Sets the {@link TlsEngineType} that will be used for processing TLS connections.
```

Reply:
I should be more diligent on the docs. Thank you 🙇